streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -437,7 +438,7 @@ StreamTask createTask(final Consumer<byte[], byte[]> consumer,
                 cache,
                 time,
                 createProducer(taskId),
-                streamsMetrics.tasksClosedSensor);
+                streamsMetrics.taskClosedSensor);
```

Comment:
If we're going to record the close in the subsequent lines of StreamThread, can we stop passing it into StreamTask? The only thing the StreamTask does with it is to record on calls to `close()`.

In other words, it seems like we're double counting by recording on the caller and the callee side, an on balance, it seems like the caller side (which you've added) is better.

Reply:
Oops I did not see it was getting passed to the StreamTask constructor and recorded in that class. I thought it wasn't getting recorded at all...

I will remove my calls to `taskClosedSensor.record()`

Reply:
oh :/ I actually liked your approach better.

Reply:
I'll do that in a separate PR, also removing the call inside of StreamTask